awscli/examples/codeartifact/get-package-version-asset.rst

```diff
@@ -4,11 +4,11 @@ The following ``get-package-version-asset`` example retrieves the package.tgz as
 
     aws codeartifact get-package-version-asset \
         --domain test-domain \
-        --repo test-repo \
+        --repository test-repo \
         --format npm \
         --package test-package \
         --package-version 4.0.0 \
-        outfileName
+        --asset outfileName
```

Comment:
I think we need to keep the `outfileName` positional arg per the docs: https://docs.aws.amazon.com/cli/latest/reference/codeartifact/get-package-version-asset.html as that is filename for downloading the asset. We also still need the `--asset` parameter but that would be the name asset hosted in CodeArtifact.